card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardView.kt

```diff
@@ -425,6 +428,15 @@ class CardView @JvmOverloads constructor(
                 binding.textInputLayoutExpiryDate.showError(localizedContext.getString(expiryDateValidation.reason))
             }
         }
+        binding.editTextExpiryDate.setContentDescription()
+    }
+
+    // TODO - Investigate contentDescription in ViewFieldState [COSDK-490]
+    private fun ExpiryDateInput.setContentDescription() {
```

Comment:
Just tested it with TalkBack and it doesn't seem to work. [Here](https://support.google.com/accessibility/android/answer/6378120?sjid=10008584266951357526-EU) it says that `contentDescription` might be unstable when set on an `EditText`.  I think to make it work we either have to add the format to the `hint` of the `TextInputLayout` or we need to add it as a helper text, like below:

![image](https://github.com/user-attachments/assets/ee6f2981-d9d9-44c9-8bfe-765cece2409b)

Also TalkBack isn't able to pronounce "MM/YY" correctly, instead it says "M-Y-Y". Is this acceptable or should we make it say the format in full?